File: fbsetup.js

const engine = require("./engine");

// module.exports = (app) => {
//     app.get('/setup', (req, res) => {
//         let mode= req.query["mode"];
//         if (mode == "get_started") {
//             engine.sendProfile();
//         } else if (mode == "persistent_menu"){
//             engine.sendProfile(persistent_menu());
//         } else {
//             engine.renderPlain(res, "Unknown setup mode");
//         }
//     });
// }

function persistent_menu(){
    return {
    "persistent_menu":[
        {
        "locale":"default",
        "composer_input_disabled": true,
        "call_to_actions":[
            {
            "type":"web_url",
            "title":"Часто задаваемые вопросы",
            "url":"https://kravkombot.herokuapp.com/groups",
            "webview_height_ratio":"tall",
            "messenger_extensions": "true"
            }
        ]
        }
    ]
    }
}

function greeting() {
  return {
        "greeting":[{
            "locale":"default",
            "text":"Здравствуйте, {{user_first_name}}, мы будем рады ответь Вам на любой вопрос." +
            " Так же Вы можете просмотреть ответы на часто задаваемые вопросы" +
            ", использовав контекстное меню чата (иконка слева от поля ввода)"
        }]
    }
}

let get_started = {"get_started":{"payload":"GREETING"}}

// engine.sendToProfile(get_started);
// engine.sendToProfile(persistent_menu());
engine.sendToProfile(greeting());
// engine.deleteFromProfile(["get_started"]);

File: fbsetup.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import requests
import json

PAGE_ACCESS_TOKEN = "EAAfxssAeEOgBAIjIhjnCEZBQAVOc9SsKg2lFjqdI8aEH6sY9ZAapkPGlV2Kb8XeZCKVOCq7P9JgtXw9ZAvMbZC8SjNqGLHYHyuCOohOAsf2lTYCaSmZBDpIXKf8KJbff9VWdIPNN0I0dgUVE1mQUPupvYNVlC8IAev4uNKNZAgWXEtWEPUmYfmR"
URL = 'https://graph.facebook.com/v3.3/me/messenger_profile?access_token=%s' % PAGE_ACCESS_TOKEN


def post(data):
    headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
    return requests.post(URL, data=json.dumps(data), headers=headers)

def get_started():
    return post({"get_started":{"payload":"GREETING"}})

def persistent_menu():
    return post(
        {
        "persistent_menu":[
            {
            "locale":"default",
            "composer_input_disabled":False,
            "call_to_actions":[
                {
                "title":"Группа Крав Мага для начинающих",
                "type":"nested",
                "call_to_actions":[
                    {
                    "title":"Help",
                    "type":"postback",
                    "payload":"HELP_PAYLOAD"
                    },
                    {
                    "title":"Contact Me",
                    "type":"postback",
                    "payload":"CONTACT_INFO_PAYLOAD"
                    }
                ]
                },
                {
                "type":"web_url",
                "title":"Visit website ",
                "url":"http://www.techiediaries.com",
                "webview_height_ratio":"full"
                }
            ]
            },
        ]
        }
    )

r = persistent_menu()


print(r.status_code)
print(r.json())